plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -352,7 +357,9 @@ class ContainerServiceFormSection extends Component {
               Memory (MiB)
             </FieldLabel>
             <FieldInput
+              min="0.001"
```

Comment:
Isn's this the `min` and not the `default` value? I thought we pre-populate the form with reasonable defaults.